plugins/woocommerce-docs/example-docs/get-started/troubleshooting/what-went-wrong.md

```diff
@@ -7,3 +7,11 @@ title: What Went Wrong?
 1. Restart?
 2. Refresh?
 3. Profit!
+
+Try some different things. _Broken?_ Try something else. **Unresponsive?** Try again.
```

Comment:
Just added some more types of markdown to one of the examples for manual testing.